vlib/v/checker/tests/anon_arg_redefinition_err.out

```diff
@@ -0,0 +1,6 @@
+vlib/v/checker/tests/anon_arg_redefinition_err.vv:2:18: error: redefinition of parameter `a`
+    1 | fn main() {
+    2 |     a := fn (a int, a int) {
+      |                     ^
+    3 |     }
+    4 | }
```

Comment:
Parser tests are better to be in `vlib/v/parser/tests/`, so that filtering with
`VTEST_ONLY=parser/tests  ./v vlib/v/compiler_errors_test.v` works.